test/test_cuda.py

```diff
@@ -4994,9 +4993,11 @@ def test_mempool_with_allocator(self):
         from torch.utils.cpp_extension import load_inline
 
         dummy_allocator_source = """
+        #include <torch/extension.h>
         extern "C" {
-          void* dummy_alloc(size_t size, int device, void* stream) { return nullptr; }
-          void dummy_free(void* ptr) { }
+          // Note that windows needs __declspec(dllexport): https://stackoverflow.com/a/24575865
+          TORCH_API void* dummy_alloc(size_t size, int device, void* stream) { return nullptr; }
```

Comment:
I am not sure TORCH_API will work here as it depends on the build flags. Maybe C10_EXPORT would be better?

Reply:
Thank you! Like like C10_EXPORT was the right thing to use.